site/content/_index.html

```diff
@@ -19,17 +19,17 @@
 
 {{< blocks/section >}}
 {{% blocks/feature icon="fa-lightbulb" title="Fast Encrypted Backups" %}}
-With a scalable architecture, **Kopia** can back everything up from your laptops to servers and everything in between.
+With a secure, scalable architecture, **Kopia** can back up everything from small laptops to large servers today.
 {{% /blocks/feature %}}
 
 
 {{% blocks/feature icon="fab fa-github" title="Contributions Welcome!" url="https://github.com/kopia/kopia/blob/master/GOVERNANCE.md" %}}
-We do a [Pull Request](https://github.com/kopia/kopia/pulls) contributions workflow on **GitHub**. New users are always welcome!
+We use a [Pull Request](https://github.com/kopia/kopia/pulls) contributions workflow on **GitHub**. New contributors and bug reports are always welcome!
 {{% /blocks/feature %}}
 
 
-{{% blocks/feature icon="fab fa-slack" title="Join The Conversation" url="https://slack.kopia.io" %}}
```

Comment:
Moving URL location because the old style led to the creation of a "Read More ..." link that was not a pointer to docs but instead to a Slack signup page.